docs/sources/v0.52.x/shared/experimental-timers-module.md

```diff
@@ -4,7 +4,7 @@ title: Experimental timers module admonition
 
 {{% admonition type="caution" %}}
 
-Starting on k6 `v0.50`, the experimental module `k6/experimental/timers` has been graduated, and its functionality is now available in the [`k6/timers` module](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-timers/). The `k6/experimental/timers` is deprecated and will be removed in `v0.52.0`.
+Starting on k6 `v0.50`, the experimental module `k6/experimental/timers` has been graduated, and its functionality is now available in the [`k6/timers` module](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-timers/). The `k6/experimental/timers` is deprecated and will be removed.
```

Comment:
Maybe point out here as well that it is available globally.

Reply:
addressed by https://github.com/grafana/k6-docs/pull/1651/commits/983414fb8614381f2b6d8b7f67a02f0864578c54